### DISCUSSION TITLE
Required packages for @carbon/ibmdotcom-styles@1.44.0

### DISCUSSION THREAD

**Discussion by catdragon** [carbon-design-system/carbon-for-ibm-dotcom#10230] [Q&A]
So, I have been tasked with updating a Vue application that uses @carbon/ibmdotcom-styles@1.44.0.  And every time I try to upgrade, I get (multiple) errors like:

```
Syntax Error: SassError: Can't find stylesheet to import.
   ╷
17 │ @import '/node_modules/carbon-components/scss/globals/scss/vendor/@carbon/elements/scss/type/font-family';
   │         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   ╵
  node_modules/@carbon/ibmdotcom-styles/scss/globals/_imports.scss 17:9              @import
  node_modules/@carbon/ibmdotcom-styles/scss/components/footer/_components.scss 9:9  @import
  node_modules/@carbon/ibmdotcom-styles/scss/components/footer/index.scss 8:9        @import
  /Users/bobbynichols/Development/MyService-rpt-frontend/src/components/ReslRPTCommon.vue 43:9                                                                         root stylesheet


 error  in ./src/assets/css/_carbon.scss
```

I tried to investigate several things and one thing I found is this: @carbon/ibmdotcom-styles@1.44.0 requires carbon-components: 10.58.5, but the latest version of carbon-components is 10.58.3. That is a disconnect for me.... not sure if it is a typo or what.

**Comment by kennylam** [comment]
Hi @catdragon, it looks like Renovate grabbed the highest version it found in the [registry](https://www.npmjs.com/package/carbon-components?activeTab=versions), which is in fact `10.58.6`, but using the `v10-next` tag. I agree this is a little confusing and we should be sticking to whatever `latest` is (currently `10.58.3`), but we haven't encountered any errors like this when building our application. Do you have an example to share?

**Reply by catdragon** [reply]
No I've had no errors, but it did screw with my "what the heck is going on here" sense.  Thought someone should be aware.
